web/conf/routes

```diff
@@ -43,6 +43,12 @@ GET     /api/v1/party/groups                controllers.api.v1.User.getAllGroups
 
 GET     /api/v1/autocomplete/search         controllers.api.v1.Search.getSearchAutoComplete()
 
+GET     /api/v1/autocomplete/search         controllers.api.v1.Search.getSearchAutoCompleteForDataset()
```

Comment:
@camelliazhang can you maybe change the version to /v2 for GET autocomplete/search